presto-main/src/main/java/com/facebook/presto/operator/scalar/ArraySortFunction.java

```diff
@@ -115,6 +117,56 @@ public int compare(Integer p1, Integer p2)
         return blockBuilder.build();
     }
 
+    @SqlType("array(bigint)")
+    public static Block bigintSort(@SqlType("array(bigint)") Block array)
```

Comment:
Should we proactively specialize other types as well ?

Reply:
eg. See the comment TODO on L73 and L92 -- can we do anything about those ?

Reply:
Yeah - will do also for double (hence draft lol)